taprpc/taprootassets.proto

```diff
@@ -684,6 +686,8 @@ message DecodeAddrRequest {
 }
 
 message ProofFile {
+    // The raw proof files encoded as bytes. Must be a file and not just an
+    // individual mint/transfer proof.
     bytes raw_proof = 1;
```

Comment:
It would probably be a good idea to rename this field to `proof_file_bytes` at some point. Not sure if this is the right PR for that.

Reply:
Yeah, I think this _is_ the PR for doing that.